js/reviewes.js

```diff
@@ -0,0 +1,44 @@
+const reviews = [
```

Comment:
[OPTIONAL] Kindly add 6 cards like on the guideline instead of 4.

<img width="965" alt="Screenshot 2023-02-09 at 9 51 54 PM" src="https://user-images.githubusercontent.com/25887253/217912241-ffd298fc-232b-4ed2-8c0f-b6b5e4761fbf.png">